Doc/library/plistlib.rst

```diff
@@ -133,14 +133,11 @@ The following functions are deprecated:
    This function calls :func:`load` to do the actual work, see the documentation
    of :func:`that function <load>` for an explanation of the keyword arguments.
 
-   .. note::
-
-      Dict values in the result have a ``__getattr__`` method that defers
-      to ``__getitem_``. This means that you can use attribute access to
-      access items of these dictionaries.
-
    .. deprecated:: 3.4 Use :func:`load` instead.
 
+   .. versionchanged: 3.7
+      Dict values in the result are now exact dicts.
```

Comment:
It would be better if we could mention the previous behavior (dicts with a `__getattr__` method) too.